src/reactapp/src/components/paymentMethod/components/PaymentMethodList.jsx

```diff
@@ -19,7 +19,7 @@ function PaymentMethodList({ methodRenderers }) {
       return;
     }
 
-    setFieldValue(fields.code, methodSelected);
+    await setFieldValue(fields.code, methodSelected);
```

Comment:
I never know this trick. thanks for the top :)

Reply:
I didn't either till I debugged the problem ... google was my friend